js/widgets/fixedToolbar.js

```diff
@@ -37,22 +39,23 @@
 
 return $.widget( "mobile.toolbar", $.mobile.toolbar, {
 	options: {
-		position: null,
+		position:null,
```

Comment:
we need to make header / footer an option lets re purpose the position option with values of header and footer instead of null and fixed. header or footer will make it fixed and set its position. for backcompat we will need the roles still of course

cc/ @gabrielschulhof @jaspermdegroot